src/client/auth/auth.proto

```diff
@@ -360,6 +373,25 @@ message SetACLRequest {
 
 message SetACLResponse {}
 
+
+//// OIDC API
+
+message GetOIDCLoginRequest {
+  string state = 1;
```

Comment:
On thinking about this a little, I think the server should be responsible for generating the state token (which could be included in `GetOIDCLoginResponse`). Otherwise all of our various client libraries and users who call our GRPC directly will be responsible for securely generating state tokens, which seems like a it could go badly in at least a few of those places